popgen/mcmcSelection.go

```diff
@@ -6,48 +6,55 @@ import (
 	"math"
 	"math/rand"
 	"fmt"
+	"log"
 	//DEBUG packages:
-	/*"log"
-	"os"
+	/*"os"
 	"runtime/pprof"*/)
 
+//To access debug prints, set verbose to 1 and then compile.
+var verbose int = 0
```

Comment:
this should be defined as a const

Reply:
When you use const at compile time, the computer will essentially go through your code and copy and paste a 0 (or any number you set it)and replace verbose. This does not affect the run time, but compile time is affected. This is different when you use var.